dev-python/urwidtrees/urwidtrees-1.0.3.ebuild

```diff
@@ -17,6 +17,10 @@ KEYWORDS="amd64 ~x86"
 
 RDEPEND=">=dev-python/urwid-1.1.0[${PYTHON_USEDEP}]"
 
+PATCHES=(
+	"${FILESDIR}/1.0.3-0001-Update-mock-dependency-to-install-when-necessary.patch"
```

Comment:
Please include the package name in the patch file, 
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html#file-naming